ext/mysqli/mysqli.c

```diff
@@ -227,7 +227,7 @@ static void mysqli_warning_free_storage(zend_object *object)
 /* }}} */
 
 /* {{{ mysqli_read_na */
-static int mysqli_read_na(mysqli_object *obj, zval *retval, bool quiet)
+static zend_result mysqli_read_na(mysqli_object *obj, zval *retval, bool quiet)
```

Comment:
Or wait actually, mysqli_read_t isn't changed yet... So you can't make this change